notebooks/vertex_genai/palm/palm_api_intro.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<ul><li>"specify a helper function --&gt; implement a helper function"</li><li>In <code>prompt</code>:  Maybe explain what is meant by "preamble", "suggestion", and example. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/350/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>